data/events/alerts.go

```diff
@@ -0,0 +1,233 @@
+package events
```

Comment:
Is this part of the formal (public) Tidepool data model or is it just an internal-only (never to be exposed to the user)? If public, then I'd like to review and comment, but it'll need to wait a bit until I have more time.

Reply:
> Is this part of the formal (public) Tidepool data model or is it just an internal-only (never to be exposed to the user)? If public, then I'd like to review and comment, but it'll need to wait a bit until I have more time.

Users would never need to interact with this directly, no. The events here are the mechanism by which we trigger and send alerts to care partners.

This code is being "shelved" due to our recent partner's contract cancellation.

Reply:
Does "shelved" mean merge into `master`? If so, we need to be sure this code isn't intentionally or unintentionally used.

Reply:
@darinkrauss the alert configuration objects in `alerts/config.go` are part of the public API (not the data model, specifically). AFAIK, there aren't a new data types introduced as part of this work.